custom_components/waste_collection_schedule/waste_collection_schedule/source/ecoharmonogram_pl.py

```diff
@@ -0,0 +1,85 @@
+import datetime
+from ..collection import Collection
+import requests
+
+DESCRIPTION = "Source for ecoharmonogram.pl"
+URL = "ecoharmonogram.pl"
+TEST_CASES = {
+    "TestName": {"town_input": "Krzeszowice", "street_input": "Wyki", "house_number_input": ""}
+}
+TITLE = "ecoharmonogram.pl"
+
+headers = {
+    'Content-Type': 'application/json; charset=utf-8',
+    'Accept': 'application/json',
+}
+
+towns_url = "https://ecoharmonogram.pl/api/api.php?action=getTowns"
+scheduled_periods_url = "https://ecoharmonogram.pl/api/api.php?action=getSchedulePeriods"
+streets_url = "https://ecoharmonogram.pl/api/api.php?action=getStreets"
+schedules_url = "https://ecoharmonogram.pl/api/api.php?action=getSchedules"
+
+
+class Source:
+    def __init__(self, town_input, street_input, house_number_input):
```

Comment:
I would omit `_input` in the arguments.

Reply:
I would rather not like to remove `_input` prefix as in fetch method I am using variable names like `town`, `streets`, that may hard later on to figure out which is input and which is town object from the API response.

Reply:
Your argumentation is ok from developer perspective, but as a user I don't want to be bothered with this details. Also other sources don't use this convention. Therefore, please remove the _input from the arguments in the constructur (you can leave the internal variable names as they are - we don't have strict conventions for that - although most python code uses the prefix underscore).

Reply:
Ok, I can see it now. The issue was about consistent API between sources. Applied the fix. Thanks.